Makefile

```diff
@@ -29,7 +29,9 @@ ifeq "$(TAG_SQLITE3)" ""
 	@echo "Missing dqlite, run \"make deps\" to setup."
 	exit 1
 endif
-
+	@go version > /dev/null 2>&1 || { echo "go is not installed for lxd-doc installation."; exit 1; }
```

Comment:
I don't know enough about Go to really review this part, but the other steps for this target require Go as well. Do they already have a check whether it's installed? Or if not, should this be made more generic?

Reply:
You're right, there is already an implicit check (line 10 with `GOPATH ?= $(shell go env GOPATH)`). I'll remove my useless check.